conda_build/source.py

```diff
@@ -336,7 +336,12 @@ def apply_patch(src_dir, path):
     try:
         check_call([patch] + patch_args, cwd=src_dir)
     except CalledProcessError:
-        sys.exit(1)
+        # fallback to -p1, the git default
+        patch_args[0] = '-p1'
```

Comment:
This feels a little fragile (not too bad) because if the arg order is changed somehow. This could start breaking. Could we find another way to change this.

Also, I don't think we are going to hit this `except` because patch will wait for user input current. Can we add an argument to make sure patch errors out.